### PR TITLE
Update: Knative compatible & versioning

### PR DESCRIPTION
**Pull request by kylegani**

### CODE REVIEW

File: cerebrium/deployments/long-running-tasks.mdx
Comment (kylegani) [inline cerebrium/deployments/long-running-tasks.mdx:1]: This URL has changed

File: v3/examples/comfyUI.mdx [hunk 1]
```diff
@@ -188,7 +194,7 @@ Once your ComfyUI application has been deployed successfully, you should be able
 ```curl
 curl --location 'https://run.cerebrium.ai/v3/p-xxxx/comfyui/predict' \
```
Comment (kylegani) [inline v3/examples/comfyUI.mdx:195]: This URL has changed

File: v4/examples/mistral-vllm.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Below, we will use the Whisper model from OpenAI to convert the video audio to text. We will then split the text into its phrase segments with its respective timings, so we know the exact source of where our model got the answer from.
+
+```python
+import torch
+from vllm import LLM, SamplingParams
+
+llm = LLM(model="mistralai/Mistral-7B-Instruct-v0.1", dtype="bfloat16")
+
+def predict(prompt, temperature, top_p, top_k, max_tokens, frequency_penalty, run_id):
```
Comment (kylegani) [inline v4/examples/mistral-vllm.mdx:60]: These are positional arguments so all of them become required, which is in conflict with the pydantic model. You'll get an error when trying to call the endpoint if you don't provide them. Structure of params needs to change (To contain defaults) and so does the pydantic model (to not include defaults)

File: v4/examples/sdxl.mdx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+name = "sdxl"
+python_version = "3.10"
+include = "[./*, main.py, cerebrium.toml]"
+exclude = "[./.*, ./__*]"
+cuda_version = "12"
+
+[cerebrium.hardware]
+region = "us-east-1"
+provider = "coreweave"
+gpu = "AMPERE_A5000"
```
Comment (kylegani) [inline v4/examples/sdxl.mdx:54]: A500s are coreweave specific

File: v4/examples/segment_anything.mdx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+We can then deploy our model to an AMPERE_A5000 instance with the following line of code
+
+```bash
+cerebrium deploy --name segment-anything --gpu AMPERE_A5000
+```
+
+After a few minutes, your model should be deployed and an endpoint should be returned. Let us create a CURL request to see the response
+
+```
+curl --location --request POST 'https://api.cortex.cerebrium.ai/v4/p-xxxxxx/segment-anything/predict' \
```
Comment (kylegani) [inline v4/examples/segment_anything.mdx:203]:
```suggestion
curl --location --request POST 'https://api.cortex.cerebrium.ai/v4/p-<YOUR PROJECT ID>/segment-anything/predict' \
```
For consistency across documentation, it's very easy to miss.

File: v4/examples/segment_anything.mdx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+1. We import the various packages we need and add the necessary ones to our **requirements.txt**
+2. We create a function that will download the image from an url if the user supplied one.
+3. We run our "Segment Anything" model with the image the user sent through the request.
+4. Lastly, we responded with the converted images. All returns from your endpoint function must be of type list or JSON.
+
+We can then deploy our model to an AMPERE_A5000 instance with the following line of code
+
+```bash
+cerebrium deploy --name segment-anything --gpu AMPERE_A5000
```
Comment (kylegani) [inline v4/examples/segment_anything.mdx:197]: A5000s are specific to coreweave

**Review by kylegani — COMMENTED** [review]
Documentation still needs a bit of work to ensure that it's:
- Not coreweave specific
- Actually works
- Updates signatures & pydantic models correctly

File: v3/examples/comfyUI.mdx (hunk 1 above)
Comment (milo157) [reply v3/examples/comfyUI.mdx:195]: this is v3 examples?

**Review by milo157 — APPROVED** [review]

**Pull request merged**